spec/19-grammar.md

```diff
@@ -3,6 +3,8 @@
 ## General
 
 The grammar notation is described in [Grammars section](09-lexical-structure.md#grammars).
+PHP scripts are encoded as ASCII, but bytes 0x80 to 0xFF are allowed in some places, as defined in the grammar. 
```

Comment:
I think the precise way to specify this would be to say that PHP scripts are encoded using an ASCII-compatible encoding. In this case we can also keep the existing `U+00XX` notation, as for an ASCII-compatible encoding `U+00XX` for `0xXX < 0x80` will always be encoded as a `0xXX` byte.

Reply:
I think the `U+00XX` notation should be avoided, as this notation is usually used to refer to Unicode codepoints. 
If this is an ASCII-compatible encoding of Unicode, then how is `U+0100` encoded? 
And from different side - ISO-8859-2 is ASCII-compatible, but Unicode codepoints are meaningless in context of that encoding.

And sorry that I'm nitpicking on the details, but I believe that as a specification, this document should be as precise as it can be.

Reply:
Per my most recent comment on the Issue, I think a reasonable wording would be something like:

> PHP scripts are parsed as a series of 8-bit bytes, rather than code points from Unicode or any other character repertoire. Within this specification, bytes are represented by their ASCII interpretations where these are printable characters.

Reply:
I wouldn't go with "printable characters", as at least according to wikipedia everything below 0x20 isn't printable, which includes the tabulation character for example.

Reply:
I'm talking about characters that will be represented literally as ASCII characters in the specification; nobody's going to type a literal tab in the middle of a markdown file to represent the byte 0x09.

Reply:
Oh, right, my bad, seems like indeed the non-printable characters always have their byte-values specified.